expression/builtin_arithmetic_vec_test.go

```diff
@@ -35,7 +35,10 @@ var vecBuiltinArithmeticCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETDecimal, childrenTypes: []types.EvalType{types.ETDecimal, types.ETDecimal}},
 		{retEvalType: types.ETDecimal, childrenTypes: []types.EvalType{types.ETDecimal, types.ETDecimal}, geners: []dataGenerator{nil, &rangeDecimalGener{0, 0, 0.2}}},
 	},
-	ast.IntDiv: {},
+	ast.IntDiv: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt}, geners: []dataGenerator{nil, &rangeInt64Gener{-10000, 10000}}},
```

Comment:
Please add more test cases to cover all four branches in your `switch-case`.
You can use `childrenFieldTypes` instead of `childrenTypes` to set signed or unsigned flags.